webcompat/templates/contributors.html

```diff
@@ -2,80 +2,30 @@
-              <span class="wc-Icon wc-Icon--contributors wc-Icon--contributors--active wc-Icon--plus" aria-hidden="true"></span>
-            </button>
-          </h2>
-        </div>
-      </div>
-      <div class="contributors__item__content wc-UIContent is-open">{% include "contributors/reproduce-bug.html" %}</div>
-    </div>
-    <div class="contributors__item">
-      <div class="contributors__item__header">
-        <div class="wc-UIContent contributors__item__title">
-          <h2 id="diagnose">
-            <button class="contributors__item__btn r-ResetButton" type="button">
-              How to Diagnose a Bug
-              <span class="wc-Icon wc-Icon--contributors wc-Icon--contributors--disabled wc-Icon--minus" aria-hidden="true"></span>
-              <span class="wc-Icon wc-Icon--contributors wc-Icon--contributors--active wc-Icon--plus" aria-hidden="true"></span>
-            </button>
-          </h2>
-        </div>
-      </div>
-      <div class="contributors__item__content wc-UIContent">{% include "contributors/diagnose-bug.html" %}</div>
-    </div>
-    <div class="contributors__item">
-      <div class="contributors__item__header">
-        <div class="wc-UIContent contributors__item__title">
-          <h2 id="outreach">
-            <button class="contributors__item__btn r-ResetButton" type="button">
-              How to Do Site Outreach
-              <span class="wc-Icon wc-Icon--contributors wc-Icon--contributors--disabled wc-Icon--minus" aria-hidden="true"></span>
-              <span class="wc-Icon wc-Icon--contributors wc-Icon--contributors--active wc-Icon--plus" aria-hidden="true"></span>
-            </button>
-          </h2>
-        </div>
-      </div>
-      <div class="contributors__item__content wc-UIContent">{% include "contributors/site-outreach.html" %}</div>
-    </div>
-    <div class="contributors__item">
-      <div class="contributors__item__header">
-        <div class="wc-UIContent contributors__item__title">
-          <h2 id="contribute">
-            <button class="contributors__item__btn r-ResetButton" type="button">
-              Contribute to this site
-              <span class="wc-Icon wc-Icon--contributors wc-Icon--contributors--disabled wc-Icon--minus" aria-hidden="true"></span>
-              <span class="wc-Icon wc-Icon--contributors wc-Icon--contributors--active wc-Icon--plus" aria-hidden="true"></span>
-            </button>
-          </h2>
+  {% include "shared/nav.html" %}
+  <main role="main">
+    <section class="grid grid-show-gap">
+      <div class="grid-row">
+        <aside class="grid-cell x1-fix">
+          <ol class="sub-nav" class="sub-nav">
+            <li class="sub-nav-header">Table of Contents</li>
+            <li class="sub-nav-active"><a href="/contributors">Join Us</a></li>
+            <li>
+              Get Started
+              <ol class="nested-sub-nav">
+                <li><a href="/contributors/reproduce-bug">Reproduce a Bug</a></li>
```

Comment:
Please add `title`s to the `<a>` to the navigation. You can ignore the actual site contents, since those will get rewritten anyway, but the nav links should have titles. :)

Reply:
Thx for the feedback. 😊 Can you explain why the nav should have titles? Afaik for assistive technology it doesn't bring any benefit for links. For sighted users it's the same as they see the link text.

Sites that deal with a11y also show that there are no titles included:
https://www.paciellogroup.com/
https://inclusive-components.design/

And quick google research returns this: https://silktide.com/i-thought-title-text-improved-accessibility-i-was-wrong/. :)

Reply:
+1 on the title (in general, not as accessibility issue). Also happy to push a fix here. :)

Reply:
if the title on the anchor is the same as the content of the anchor. It is not very useful and probably even annoying for non-sighted users.